detection_rules/schemas/v7_12.py

```diff
@@ -31,7 +31,7 @@ class ThresholdCardinality(jsl.Document):
             field = jsl.StringField(required=True)
             value = jsl.IntField(minimum=1, required=True)
 
-        field = jsl.ArrayField(jsl.StringField(required=True, default=""))
+        field = jsl.ArrayField(jsl.StringField(required=False, min_length=1), required=True)
```

Comment:
this is gonna have a merge conflict with #1084.
but that's easy, we can fix it there

Reply:
i think the required is confusing inside of the array. i'm not sure what what even means
```suggestion
        field = jsl.ArrayField(jsl.StringField(min_length=1), required=True)
```